src/lib/toolbar.ts

```diff
@@ -681,6 +680,10 @@ export const toolbarConfig: Record<ToolbarModeName, ToolbarMode> = {
             title: 'Center rectangle',
             description: 'Start drawing a rectangle from its center',
             links: [],
+            hotkey: (state) =>
+              state.matches({ Sketch: 'Center Rectangle tool' })
+                ? ['Esc', 'Alt+R']
```

Comment:
Just a dumb question probably: why is the order here flipped vs Circle three point tool?

Reply:
No reason at all let me fix
```suggestion
                ? ['Alt+R', 'Esc']
```